assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -165,12 +173,15 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
       </Body>
       <Footer>
         {/* TODO: implement (see https://github.com/google/web-stories-wp/issues/262) */}
-        <EyedropperButton
-          width={EYEDROPPER_ICON_SIZE}
-          height={EYEDROPPER_ICON_SIZE}
-          aria-label={__('Select color', 'web-stories')}
-          isDisabled
-        />
+        {enableEyeDropper ? (
+          <EyedropperButton
+            width={EYEDROPPER_ICON_SIZE}
+            height={EYEDROPPER_ICON_SIZE}
+            aria-label={__('Select color', 'web-stories')}
+          />
+        ) : (
+          <span />
```

Comment:
What's that for? Let's not render unnecessary elements.

Could do `enableEyeDropper  && <EyedropperButton  ....>`